cmd/req.go

```diff
@@ -110,6 +114,7 @@ func (r *reqCommand) Setup() (err error) {
 	r.cmd.Flag("verbose", "Enable verbose output").Short('v').BoolVar(&r.verbose)
 	r.cmd.Flag("display", "Display only a subset of results (ok, failed, all, none)").EnumVar(&r.displayOverride, "ok", "failed", "all", "none")
 	r.cmd.Flag("discovery-timeout", "Timeout for doing discovery").PlaceHolder("SECONDS").IntVar(&r.discoveryTimeout)
+	r.cmd.Flag("output-file", "Filename to write output to").PlaceHolder("FILENAME").Short('o').StringVar(&r.outputFile)
```

Comment:
you can use `FileVar` or `OpenFileVar` and it will open it for you if supplied etc, granted its not a HUGE improvement but a bit I guess.

Reply:
I still need to check if the var was used, so I just elected to keep as is, if that's ok